yt/chyt/controller/internal/jupyt/controller.go

```diff
@@ -17,6 +16,26 @@ const (
 )
 
 type Config struct {
+	YtAuthCookieName *string           `yson:"yt_auth_cookie_name"`
+	ExtraEnvVars     map[string]string `json:"extra_env_vars"`
```

Comment:
`yson`, not `json`

Reply:
Oh, my bad, copilot is need to be double checked:(